tools/wake/describe.cpp

```diff
@@ -182,6 +182,23 @@ void describe(const std::vector<JobReflection> &jobs, bool script, bool debug, b
   }
 }
 
+void describe_failed(const std::vector<JobOutput> &jobs) {
```

Comment:
describe failed should just be "describe_userfacing" I think.

Reply:
This will mean it needs to take JobReflection and that JobReflection will ned to be tweaked slightly along with other users of JobRefleciton in the describe.cpp file